.cirrus.yml

```diff
@@ -95,11 +96,33 @@ gating_task:
         - '/usr/local/bin/entrypoint.sh lint'
 
 
+build_each_commit_task:
+
+    depends_on:
+        - "gating"
+
+    gce_instance:
+        image_project: "libpod-218412"
+        zone: "us-central1-a"  # Required by Cirrus for the time being
+        cpu: 2
+        memory: "4Gb"
+        disk: 40
+        matrix:
+            image_name: "fedora-29-libpod-0c954a67"
+
+    timeout_in: 20m
+
+    script:
+        - $SCRIPT_BASE/setup_environment.sh
+        - git fetch --depth $CIRRUS_CLONE_DEPTH origin $CIRRUS_BASE_BRANCH
+        - env GOPATH=/var/tmp/go/ make build-all-new-commits GIT_BASE_BRANCH=origin/$CIRRUS_BASE_BRANCH
```

Comment:
Hmmm, `setup_environment.sh` exports GOPATH, odd you need to set it.  Must be something to do with the way cirrus executes the commands.

If/When you move these steps into their own little script (with a loop), you should find the environment is properly setup (including `$GOPATH`).  I vaguely remember something like this happening to me before.